src/vue/pages/VerificationCorporate.vue

```diff
@@ -210,7 +210,17 @@ export default {
   computed: {
     ...mapGetters({
       kvEntryCorporateRoleId: vuexTypes.kvEntryCorporateRoleId,
+      isAccountRoleReset: vuexTypes.isAccountRoleReset,
+      previousAccountRole: vuexTypes.kycPreviousAccountRole,
     }),
+    isFormEditable () {
+      return this.isAccountRoleReset ||
+        this.kycState === REQUEST_STATES_STR.rejected
+    },
+    isFormAutocompleted () {
+      return (this.previousAccountRole === this.kvEntryCorporateRoleId) ||
```

Comment:
what about general

Reply:
It is handled in general form, we don't need to fill the corporate form in that case